.github/workflows/test-past-releases.yml

```
@@ -0,0 +1,168 @@
+name: Test past releases
+
+on:
+  schedule:
+    # https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onschedule
+    # > Scheduled workflows run on the latest commit on the default or base branch
+    # i.e. this can only run on master
+    - cron:  '0 11 * * *'
```

I agree with your suggestion of running this weekly, but I think the current cron line says "11am UTC every day" (that is, 6am EST or 7am EDT). I believe this would limit it to weekly on Sundays (presumably a not-busy day for us):
```suggestion
    - cron:  '0 11 * * 0'
```